test/tap/search.js

```diff
@@ -358,6 +184,7 @@ function setup () {
 }
 
 function cleanup () {
+  server.done()
```

Comment:
close shuts down the server. done does missing-request checks while letting us reuse the same server (so, it errors if we don't hit the mocked endpoint at all)